docs/CONTRIBUTING.md

```diff
@@ -55,22 +57,19 @@ To contribute code to ODK Collect, you will need to open a [pull request](https:
 
 1. Keep your pull request focused on one narrow goal. This could mean addressing an issue with multiple, smaller pull requests. Small pull requests are easier to review and less likely to introduce bugs. If you would like to make stylistic changes to the code, create a separate pull request.
 
-1. Run `./gradlew checkCode` and fix any errors.
```

Comment:
This is part of the PR template and doesn't feel to me like it needs to be highlighted here.